src/views/Search.vue

```diff
@@ -27,57 +27,60 @@
       </header>
 
       <div class="col-xs-12 col-md-9">
-        <div class="alert alert-dismissible alert-warning">FIXME</div>
         <b-pagination
-          :total-rows="100"
-          v-model="page"
-          :per-page="25"
-          size="md"/>
+          :total-rows="numFound"
+          :per-page="rowsPerPage"
+          v-model="currentPage"
+          responsive="true"
+          class="table-sm table-border-soft mt-2"
+          size="md"
+        />
         <div class="search-results-rows">
-          <table
-            :id="'selenium_id_' + selenium_id"
-            class="search-results-table table table-striped table-sm simpletable">
-            <thead>
-              <tr>
-                <th width="25%">Term</th>
-                <th width="15%">Category</th>
-                <th width="25%">Taxon</th>
-                <th>Matching String</th>
-              </tr>
-            </thead>
-            <tbody>
-              <tr
-                v-for="(result, index) in searchResults"
-                :key="index"
-                class="search-result-item">
-                <td>
-                  <router-link :to="result.toLink">
-                    {{ result.label }}
-                  </router-link>
-                </td>
-                <td>{{ result.category }}</td>
-                <td>{{ result.taxon }}</td>
-                <td v-html="result.highlight"/>
-              </tr>
-            </tbody>
-          </table>
+
+          <div v-if="searchResults && searchResults.length > 0 ">
+            <h3><span class="searchTerm">{{ query }}</span>  has <span class="searchTerm">{{ numFound }}</span> matches</h3>
+            <b-table
+              :fields="fields"
+              :items="rowsProvider"
+              :current-page="currentPage"
+              :per-page="rowsPerPage"
+              striped
+              responsive="true"
+              class="table-sm table-border-soft"
+              hover>
+              <template
+                slot="label"
+                slot-scope="row"
+              >
+                <router-link :to="row.item.toLink">
+                  {{ row.item.label }}
+                </router-link>
+              </template>
+              <template
+                slot="highlight"
+                slot-scope="row"
+              >
+                <span v-html="row.item.highlight"/>
+              </template>
+            </b-table>
+          </div>
         </div>
-      </div>
 
 
-      <footer class="footer">
-        <home-footer/>
-      </footer>
+        <footer class="footer">
+          <home-footer/>
```

Comment:
I mentioned this before somewhere (Gitter or another PR). Please use `FooterAll.vue` instead of `HomeFooter.vue`. FooterAll.vue is intended to be a thinner footer than that on the home page.